nuancier/lib/model.py

```diff
@@ -362,7 +362,7 @@ def get_results(cls, session, election_id):
             Candidates.date_created,
             Candidates.date_updated
         ).order_by(
-            'votes DESC'
+            sa.text('votes DESC')
```

Comment:
Is this really needed or it is just a good habit?

Reply:
it seems sqlalchemy is now making this error (it was one of the main reason travis was failing)